src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -215,12 +216,32 @@ public function exportAction(OrderFilters $filters)
             ->setFileName('order_' . date('Y-m-d_His') . '.csv');
     }
 
+    public function getPreviewDataAction(int $orderId): Response
+    {
+        try {
+            /** @var OrderPreview $orderPreview */
+            $orderPreview = $this->getQueryBus()->handle(new GetOrderPreview($orderId));
+
+            return $this->json([
+                'preview' => $this->renderView('@PrestaShop/Admin/Sell/Order/Order/preview.html.twig', [
+                    'orderPreview' => $orderPreview,
+                ]),
+            ]);
+        } catch (Exception $e) {
```

Comment:
there are million reasons to catch unintended Exception here sometimes hiding true errors which should be displayed in any case. Can you catch here more predictable and specific exception or multiple exceptions if such occur? :thinking:

Reply:
Yes it's related to modules throwing errors.

Let's leave this like it for now, as this issue is global and has been raised by @sarjon in https://github.com/PrestaShop/PrestaShop/issues/15732. Following the solution we'll choose for https://github.com/PrestaShop/PrestaShop/issues/15732 we'll apply it everywhere